blog/2023-12-12-nushell_0_88_0.md

```diff
@@ -80,6 +80,45 @@ Thanks to all the contributors below for helping us making the documentation of
 | ------------------------------------ | ----------- | ------------------------------------------------------- |
 | [@author](https://github.com/author) | ...         | [#12345](https://github.com/nushell/nushell/pull/12345) |
 
+## Spread operator
+
+[#10598](https://github.com/nushell/nushell/issues/10598) asked for a spread operator (`...`), and [#11006](https://github.com/nushell/nushell/pull/11006)
```

Comment:
if you're ok with that, please add your name somewhere here, as a link to your GitHub account.
you are the one that made that possible, you should be in the spotlight with the spread operator next release :wink: :heart:

Reply:
Always happy to toot my own horn :)

Reply:
perfect :yum: